### PR TITLE
Few suggestions from Tanya 

### PR DESCRIPTION
**Pull request by tanyapowell**
Hola 👋 

Few suggestions on extensions I'm a fan of:
### [Prettier](https://github.com/prettier/prettier)
It's an opinionated code formatter to enforces a consistent style. It can work with or without ESLint to adhere to the projects code styling, and my fave part of is it can format on save/paste 🙌 

### [Formatting Toggle](https://github.com/Tom-Bonnike/vscode-formatting-toggle)
If the project you are working on doesn't have linting or prettier, this extension can toggle your prettier extension on and off with the simple click of a button. It's saved me a few times!

Couldn't have said it better myself

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,10 @@
         "wayou.vscode-todo-highlight",
         "wix.vscode-import-cost",
         "YoshinoriN.current-file-path",
-        "Zignd.html-css-class-completion"
+        "Zignd.html-css-class-completion",
+        "esbenp.prettier-vscode",
+        "jkjustjoshing.vscode-text-pastry",
+        "tombonnike.vscode-status-bar-format-toggle"
```
Comment (eddiejaoude) [inline package.json]: It’s not alphabetical 😱😜

**Comment by eddiejaoude** [comment]
They look great @tanyapowell , thank you 👍🤓

File: README.md [hunk 2]
```diff
@@ -42,6 +42,9 @@ https://marketplace.visualstudio.com/items?itemName=eddiejaoude.js-fullstack-ext
 - [Javascript console utils](https://marketplace.visualstudio.com/items?itemName=whtouche.vscode-js-console-utils)
 - [Version Lens](https://marketplace.visualstudio.com/items?itemName=pflannery.vscode-versionlens)
 - [Import cost](https://marketplace.visualstudio.com/items?itemName=wix.vscode-import-cost)
+- [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+- [Text Pastry](https://marketplace.visualstudio.com/items?itemName=jkjustjoshing.vscode-text-pastry)
```
Comment (eddiejaoude) [inline README.md]: These 3 are for multiple languages, may be best in General section?

File: package.json (hunk 1 above)
Comment (tanyapowell) [reply package.json]: 🙈

File: README.md (hunk 2 above)
Comment (tanyapowell) [reply README.md]: Thought ESLint is Javascript specific? I know Prettier supports typescript, but thought ESLint was only for JS 🤔

File: package.json (hunk 1 above)
Comment (eddiejaoude) [reply package.json]: Its not a big deal, I like to start of organised, I might not end up there in a few months 😂

File: README.md (hunk 2 above)
Comment (eddiejaoude) [reply README.md]: hmm I think you are right 🤓

**Review by eddiejaoude — APPROVED** [review]

**Comment by eddiejaoude** [comment]
Thanks @tanyapowell 👍🤓

**Pull request merged**